src/main/com/yetanalytics/datasim.clj

```diff
@@ -15,29 +15,31 @@
 
 ;; TODO: Add fdefs for generation functions
 ;; TODO: Extract more implementation code from `sim` namespace
-;; TODO: Use more opt-map passing (instead of `apply`) in Clojure 1.11
 
+#_{:clj-kondo/ignore [:unused-binding]}
 (defn generate-seq
   "Given `input`, produce a lazy sequence of statements in a synchronous
    fashion."
-  [input & kwargs]
-  (apply sim/sim-seq input kwargs))
+  [input & {:keys [select-agents] :as kwargs}]
```

Comment:
So the unused bindings are basically for documentation purposes? I don't hate that.

Reply:
Yep, which is pretty important for the top-level API functions.